src/Mvc/Mvc.TagHelpers/src/ComponentTagHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Mvc.ViewFeatures.RazorComponents;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers
+{
+    /// <summary>
+    /// A <see cref="TagHelper"/> that renders a Blazor component.
+    /// </summary>
+    [HtmlTargetElement("component", Attributes = ComponentTypeName)]
```

Comment:
should we enforce the style here?

Reply:
"Razor component" vs. "Blazor component"?
Technically, as it is right now, they're Razor Components.